packages/react-ui-core/src/Ratings/index.js

```diff
@@ -0,0 +1,4 @@
+export { default as RatingBar } from './RatingBar'
+export { default as Star } from './SampleSvgs/Star'
+export { default as Square } from './SampleSvgs/Square'
```

Comment:
Square and Circle do *not* belong in `src` code. They should only be included in storybook.